File: comando.py
import signal, sys, os, time, serial

def comando(com, app):

	linea = os.popen('tail -1 /home/pi/tg/datos.txt').read()

	if com=="Encender led 1":
		linea='1'+linea[1:]
		fichero = open('/home/pi/tg/datos.txt', 'w')
		fichero.write(linea)
		fichero.close()

	if com=="Encender led 2":
		linea=linea[:2]+'1'+linea[3:]
		fichero = open('/home/pi/tg/datos.txt', 'w')
		fichero.write(linea)
		fichero.close()

	if com=="Apagar led 1":
		linea='0'+linea[1:]
		fichero = open('/home/pi/tg/datos.txt', 'w')
		fichero.write(linea)
		fichero.close()

	if com=="Apagar led 2":
		linea=linea[:2]+'0'+linea[3:]
		fichero = open('/home/pi/tg/datos.txt', 'w')
		fichero.write(linea)
		fichero.close()

	fichero2 = open('/home/pi/tg/registro.txt', 'a')
	fichero2.write(app)
	fichero2.write(" ")
	fichero2.write(com)
	fichero2.write(" -- ")
	fichero2.write(time.strftime("%c"))
	fichero2.write("\n")
	fichero2.close()
	return

def arduino(com, app):
	fichero = open('/home/pi/tg/ir.txt', 'w')
        fichero.write(com)
        fichero.close()
	
	fichero2 = open('/home/pi/tg/registro.txt', 'a')
        fichero2.write(app)
        fichero2.write(" ")
        fichero2.write(com)
        fichero2.write(" -- ")
        fichero2.write(time.strftime("%c"))
        fichero2.write("\n")
        fichero2.close()
	return

File: comando.php
<?php
	$parametro=$_POST['parametro'];
	$linea=shell_exec("tail -1 /home/pi/tg/datos.txt");
        $datos=explode(" ", $linea);
	if(strcmp($parametro, "Encender led 1")==0)
        	$datos[0]="1";
	if(strcmp($parametro, "Apagar led 1")==0)
                $datos[0]="0";
        if(strcmp($parametro, "Encender led 2")==0)
                $datos[1]="1";
        if(strcmp($parametro, "Apagar led 2")==0)
                $datos[1]="0";

	$linea="";
        for($i=0; $i<count($datos); $i++)
        {
		$linea.=$datos[$i];
		if($i<count($datos)-1)
			$linea.=" ";
        }
	$fichero = fopen("/home/pi/tg/datos.txt", "w");
	$flag=fwrite($fichero, $linea);
	fclose($fichero);

	date_default_timezone_set('Europe/Madrid');
	$parametro="(web) ".$parametro." -- ".date("D M j G:i:s")."\n";
	$fichero2 = fopen("/home/pi/tg/registro.txt", "a");
        $flag=fwrite($fichero2, $parametro);
        fclose($fichero2);

	echo $parametro;
?>
